res/lang/stations/stations_en.txt

```diff
@@ -22,7 +22,7 @@ MAP_COUNTRY_DE-NI_LONG  = Lower Saxony
 MAP_COUNTRY_DE-SH_LONG  = Schleswig-Holstein
 MAP_COUNTRY_DE-BB_LONG  = Brandenburg
 MAP_COUNTRY_DE-NW_LONG  = North Rhine-Westphalia
-MAP_COUNTRY_DE-MV_LONG  = Mecklenburg-Hither Pomerania
+MAP_COUNTRY_DE-MV_LONG  = Mecklenburg-Far Pomerania
```

Comment:
Seems we both were not correct then...
Mecklenburg–West Pomerania
https://www.britannica.com/place/Mecklenburg-West-Pomerania

Mecklenburg–Western Pomerania
https://en.wikipedia.org/wiki/Mecklenburg-Vorpommern

Reply:
Yeah I changed it to West shortly after pushing this revision :-)